pkg/snowflake/resource_monitor_test.go

```diff
@@ -22,14 +22,14 @@ func TestResourceMonitor(t *testing.T) {
 	a.Equal(`DROP RESOURCE MONITOR "resource_monitor"`, q)
 
 	ab := rm.Alter()
-	ab.SetInt("credit_quota", 66)
+	ab.SetFloat("credit_quota", 66.6)
 	q = ab.Statement()
-	a.Equal(`ALTER RESOURCE MONITOR "resource_monitor" SET CREDIT_QUOTA=66`, q)
+	a.Equal(`ALTER RESOURCE MONITOR "resource_monitor" SET CREDIT_QUOTA=66.600000`, q)
 
 	cb := snowflake.ResourceMonitor("resource_monitor").Create()
 	cb.NotifyAt(80).NotifyAt(90).SuspendAt(95).SuspendImmediatelyAt(100)
 	cb.SetString("frequency", "YEARLY")
-	cb.SetInt("credit_quota", 666)
+	cb.SetFloat("credit_quota", 666.66666666)
```

Comment:
i wonder if it is also worth keeping a test where the input "looks like an integer" and making sure the parsing works properly

Reply:
good point